content/applications/general/auth/azure.rst

```diff
@@ -2,9 +2,159 @@
 Microsoft Azure sign-in authentication
 ======================================
 
-Due to specific requirements in Azure's OAuth implementation, Microsoft Azure OAuth identification
-is NOT compatible with Odoo at the moment.
+The Microsoft Azure OAuth sign-in authentication is a useful function that allows Odoo users to sign
+into their database with their Microsoft Azure account.
+
+This is particularly helpful if the organization uses Azure Workspace, and wants employees within
+the organization to connect to Odoo using their Microsoft Accounts.
+
+.. warning::
+   Databases hosted on Odoo.com should not use OAuth login for the owner or administrator of the
+   database as it would unlink the database from their Odoo.com account. If OAuth is set up for that
+   user, the database will no longer be able to be duplicated, renamed, or otherwise managed from
+   the Odoo.com portal.
+
 
 .. seealso::
    - :doc:`../../productivity/calendar/outlook`
    - :doc:`/administration/maintain/azure_oauth`
+
+Configuration
+=============
+
+Integrating the Microsoft sign-in function requires configuration on Microsoft and Odoo.
+
+Odoo System Parameter
+---------------------
+
+First activate the :ref:`developer mode <developer-mode>`, and then go to :menuselection:`Settings
+--> Technical --> System Parameters`.
+
+Click :guilabel:`Create` and on the new/blank form that appears, add the following system parameter
+`auth_oauth.authorization_header` to the :guilabel:`Key` field, and set the :guilabel:`Value` to
+`1`. Then click :guilabel:`Save` to finish.
+
+Microsoft Azure dashboard
+-------------------------
+
+Create a new application
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To get started, go to `Microsoft's Azure Portal <https://portal.azure.com/>`_. Log in with the
```

Comment:
Incomplete + redundant intro since we already said "First..." in the previous section.

If anything, this section would now begin with something along the lines of "Now that the system parameters in Odoo have been set up, it's time to create a corresponding application inside of Microsoft Azure."